exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/impl/StringFunctions.java

```diff
@@ -416,16 +417,25 @@ public void setup() {
 
     @Override
     public void eval() {
-      if (index.value < 1) {
+      if (index.value == 0) {
```

Comment:
Sigh, 1-based indexing.  I think the logic you chose in this PR for non-positive index values is the best we can do, considering.

Reply:
> Sigh, 1-based indexing. I think the logic you chose in this PR for non-positive index values is the best we can do, considering.
@jnturton Hi James, anything wrong here?